Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -436,6 +449,7 @@ def setup(self):
     self.updateOutputType()
     self.updateVideoFormat(0)
     self.updateViewOptions()
+    self.updateSingleStepOptions(self.singleStepButton.checked)
```

Comment:
It would be nicer to enable volume node output also when the number of output images is set to 1 (without clicking "single" option). You can do all these checks in updateOutputType, where all kinds of checks and GUI states updates are done already (no need for a new callback function).

Reply:
The number of output images is not defined as "Single" by any other interaction currently. If the `numberOfStepsSliderWidget` value becomes 1 it does not set the "Single" option checked. Are you suggesting this should be the case? Therefore the `setNumberOfSteps` slot would call to check `singleStepButton`.

Changes to `updateOutputType` would not make sense as currently I'm not adding a new option to this combobox. It still has the options "Image Series", "Video" and "lightbox image". I don't think "Video" or "lightbox image" selection would ever enable volume node output qMRMLNodeComboBox.

Reply:
"Single" button is a useful shortcut for temporarily changing the number of request output images to 1. It has the same effect as moving "Number of images" slider to 1. We just had to add this button because many users did not realize they can create single screenshots by moving the slider to 1.

I don't mind if you keep the separate callback function. It is just a matter of balancing to avoid both too large callback functions and too many callback functions (both decrease readability of the code).

Reply:
I guess I just don't know how to handle its initial state appropriately where it also makes sense. `setNumberOfSteps` is not called at startup and it doesn't make sense to include it any of the current slots (`updateOutputType`/`updateViewOptions`).

Reply:
If you don't want a slot I could do the following in the setup
```python
self.volumeNodeComboBox.enabled = False
.
.
.
self.singleStepButton.connect('toggled(bool)', self.numberOfStepsSliderWidget, 'setDisabled(bool)')
self.singleStepButton.connect('toggled(bool)', self.volumeNodeComboBox, 'setEnabled(bool)')
```